uv.lock

```diff
@@ -0,0 +1,1195 @@
+version = 1
+requires-python = ">=3.8"
+
+[[distribution]]
+name = "annotated-types"
+version = "0.7.0"
+source = "registry+https://pypi.org/simple"
+sdist = { url = "https://files.pythonhosted.org/packages/ee/67/531ea369ba64dcff5ec9c3402f9f51bf748cec26dde048a2f973a4eea7f5/annotated_types-0.7.0.tar.gz", hash = "sha256:aff07c09a53a08bc8cfccb9c85b05f1aa9a2a6f23728d790723543408344ce89", size = 16081 }
+wheels = [{ url = "https://files.pythonhosted.org/packages/78/b6/6307fbef88d9b5ee7421e68d78a9f162e0da4900bc5f5793f6d3d0e34fb8/annotated_types-0.7.0-py3-none-any.whl", hash = "sha256:1f02e8b43a8fbbc3f3e0d4f0f4bfc8131bcb4eebe8849b8e5c773f3a1c582a53", size = 13643 }]
+
+[[distribution.dependencies]]
+name = "typing-extensions"
+version = "4.12.2"
+source = "registry+https://pypi.org/simple"
+marker = "python_version < '3.9'"
+
+[[distribution]]
+name = "anyio"
+version = "4.4.0"
+source = "registry+https://pypi.org/simple"
+sdist = { url = "https://files.pythonhosted.org/packages/e6/e3/c4c8d473d6780ef1853d630d581f70d655b4f8d7553c6997958c283039a2/anyio-4.4.0.tar.gz", hash = "sha256:5aadc6a1bbb7cdb0bede386cac5e2940f5e2ff3aa20277e991cf028e0585ce94", size = 163930 }
+wheels = [{ url = "https://files.pythonhosted.org/packages/7b/a2/10639a79341f6c019dedc95bd48a4928eed9f1d1197f4c04f546fc7ae0ff/anyio-4.4.0-py3-none-any.whl", hash = "sha256:c1b2d8f46a8a812513012e1107cb0e68c17159a7a594208005a57dc776e1bdc7", size = 86780 }]
+
+[[distribution.dependencies]]
+name = "exceptiongroup"
+version = "1.2.1"
+source = "registry+https://pypi.org/simple"
+marker = "python_version < '3.11'"
+
+[[distribution.dependencies]]
+name = "idna"
+version = "3.7"
+source = "registry+https://pypi.org/simple"
+
+[[distribution.dependencies]]
+name = "sniffio"
+version = "1.3.1"
+source = "registry+https://pypi.org/simple"
+
+[[distribution.dependencies]]
+name = "typing-extensions"
+version = "4.12.2"
+source = "registry+https://pypi.org/simple"
+marker = "python_version < '3.11'"
+
+[[distribution]]
+name = "autobot-ml"
+version = "0.0.16"
+source = "editable+file:///Users/crmarsh/workspace/autobot"
+sdist = { url = "file:///Users/crmarsh/workspace/autobot" }
```

Comment:
it exposed local path of project, which is not good for whom respect privacy and also uploads their `uv.lock` publicly.

Reply:
Correct, we're changing it this week.

Reply:
See, e.g., https://github.com/astral-sh/uv/pull/4205.